### ISSUE TITLE
whaat's the result? any result like accuracy? precison,recall and f1 value?

### ISSUE THREAD

**Issue by 652994331** [MarsRoger/albert_ner#3]
Hi, we checked your code, good job. Can you please share us the result of your experiments? like accuracy, precision,recall and f1? recently we are doing albert_bilstm+crf but for an albert_large_zh. it's nice if we can see your prediction result. Thanks so much

**Comment by MarsRoger** [comment]
![image](https://user-images.githubusercontent.com/39691448/71386292-d659a280-2627-11ea-82a7-3fbe99ea01ec.png)

**Comment by MarsRoger** [comment]
results on validation set:
total: accuracy:  98.59%; precision:  89.15%; recall:  89.29%; FB1:  89.22
in the class of LOC: precision:  90.01%; recall:  90.16%; FB1:  90.09
in the class of ORG: precision:  84.41%; recall:  87.09%; FB1:  85.73
in the class of PER: precision:  93.11%; recall:  90.36%; FB1:  91.71
results on test set:
the total: accuracy:  98.27%; precision:  86.86%; recall:  87.74%; FB1:  87.30
in the class of LOC: precision:  87.79%; recall:  87.01%; FB1:  87.40
in ORG:precision:  81.44%; recall:  85.92%; FB1:  83.62
in PER:  precision:  92.05%; recall:  91.59%; FB1:  91.82  1811
I believe you can get a better result after fine-tuning

**Comment by MarsRoger** [comment]
> Hi, we checked your code, good job. Can you please share us the result of your experiments? like accuracy, precision,recall and f1? recently we are doing albert_bilstm+crf but for an albert_large_zh. it's nice if we can see your prediction result. Thanks so much

results on validation set:
total: accuracy: 98.59%; precision: 89.15%; recall: 89.29%; FB1: 89.22
in the class of LOC: precision: 90.01%; recall: 90.16%; FB1: 90.09
in the class of ORG: precision: 84.41%; recall: 87.09%; FB1: 85.73
in the class of PER: precision: 93.11%; recall: 90.36%; FB1: 91.71
results on test set:
the total: accuracy: 98.27%; precision: 86.86%; recall: 87.74%; FB1: 87.30
in the class of LOC: precision: 87.79%; recall: 87.01%; FB1: 87.40
in ORG:precision: 81.44%; recall: 85.92%; FB1: 83.62
in PER: precision: 92.05%; recall: 91.59%; FB1: 91.82 1811
I believe you can get a better result after fine-tuning

**Comment by 652994331** [comment]
Thanks so much for the reply. Just curious, did you compare the performance of albert_base_zh and bert for the same NER mission with same parameters like learning rate , batch&nbsp;



发自我的iPhone


------------------ Original ------------------

**Comment by MarsRoger** [comment]
sry, i didn't care that.------------------ 原始邮件 ------------------
发件人: "652994331"<notifications@github.com>
发送时间: 2019年12月24日(星期二) 上午8:56
收件人: "MarsRoger/albert_ner"<albert_ner@noreply.github.com>;
抄送: "MarsRoger"<762341055@qq.com>;"Comment"<comment@noreply.github.com>;
主题: Re: [MarsRoger/albert_ner] whaat's the result? any result likeaccuracy? precison,recall and f1 value? (#3)


Thanks so much for the reply. Just curious, did you compare the performance of albert_base_zh and bert for the same NER mission with same parameters like learning rate , batch&nbsp; 
 
 
 
 发自我的iPhone 
 
 
 ------------------ Original ------------------
